spec/lib/activitypub/linked_data_signature_spec.rb

```diff
@@ -81,6 +81,6 @@ def sign(from_account, options, document)
     options_hash   = Digest::SHA256.hexdigest(canonicalize(options.merge('@context' => ActivityPub::LinkedDataSignature::CONTEXT)))
     document_hash  = Digest::SHA256.hexdigest(canonicalize(document))
     to_be_verified = options_hash + document_hash
-    Base64.strict_encode64(from_account.keypair.sign(OpenSSL::Digest::SHA256.new, to_be_verified))
+    Base64.strict_encode64(from_account.keypair.sign(OpenSSL::Digest.new('SHA256'), to_be_verified))
```

Comment:
Any specific reason for that change?

Reply:
[Here's](https://deepsource.io/gh/withshubh/mastodon/issue/RB-LI1083/description) the reason why this is an anti-pattern.

Reply:
weirdly enough, I can't find anything about this supposed deprecation, the documentation still suggests using those classes, etc.
however, the change shouldn't hurt either, behind the hoods those classes just call `OpenSSL::Digest.new(name)` anyway

Reply:
Well, Yes. The latter is the preferred way of writing as it improves the readability.